apt/method.go

```diff
@@ -66,28 +68,33 @@ type Method struct {
 
 type aptMethodConfig struct {
 	serviceAccountJSON, serviceAccountEmail string
+	debug                                   bool
 }
 
 // Run runs the method.
-func (m *Method) Run(ctx context.Context) {
+func (m *Method) Run(ctx context.Context) error {
 	m.writer.SendCapabilities()
 	for {
 		select {
 		case <-ctx.Done():
-			return
+			return nil
 		default:
 		}
 		msg, err := m.reader.ReadMessage(ctx)
-		if err != nil {
+		if err == errEmptyMessage {
 			continue
+		} else if err == io.EOF {
+			return nil
+		} else if err != nil {
+			return err
 		}
 		switch msg.code {
 		case 600:
 			m.handleAcquire(ctx, msg)
 		case 601:
 			m.handleConfigure(msg)
 		default:
-			// TODO: now write a test for this.
+			// TODO(hopkiw): now write a test for this.
```

Comment:
why the ownership on the TODO here?

Reply:
Per the style guide, every TODO message must be tied to either a username, or a bug ticket (in the context of open source code, this would presumably mean a GitHub issue number). There should **never** be a "naked" TODO in any code we write.